Website/App_GlobalResources/Exceptions.resx

```diff
@@ -211,7 +211,7 @@
     <value>Target User '{0}' has Blocked Relationship '{1}' from Target User '{2}'.</value>
   </data>
   <data name="UserRelationshipExistsError.Text" xml:space="preserve">
-    <value>Relationship already exists for Initiating User '{0}' Target User '{1}' RelationshipID '{2}'.</value>
+    <value>You are already following this user.</value>
```

Comment:
@mgatmaytan , this resource key was using a string.format values. You have changed the string removing the {} but I did not see in your commit the change in where this key is used in the code to adapt it to this change. Can you please review it? thks